packages/fxa-settings/src/components/ConfirmWithLink/mocks.tsx

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import ConfirmWithLink, { ConfirmWithLinkProps } from '.';
+import { MOCK_ACCOUNT } from '../../models/mocks';
+
+export const MOCK_EMAIL_PROVIDER = 'Email Provider';
+export const MOCK_GOBACK_CB = () => {
+  console.log('Navigating back!');
+};
+export const MOCK_RESEND_CB = () => {
+  console.log('Resending link!');
+};
+export const MOCK_STRINGS = {
+  headingFtlId: 'mock-confirmation-heading',
+  headingText: 'Confirm something',
+  instructionFtlId: 'mock-confirmation-instruction',
+  instructionText: `Open the mock link sent to ${MOCK_ACCOUNT.primaryEmail.email} ASAP`,
+};
```

Comment:
Not necessary for r+ but we do have this type in `auth-server` and could use a version of it here (or move to `fxa-shared/l10n`):

```
export interface FtlIdMsg {
  id: string;
  message: string;
}
```

So this could be like...
```
const ftlIdMsgs: FtlIdMsg[] = {
  heading: {
    ftlId: '',
    message: '',
  },
  instruction: {
    ftlId: '',
    message: '',
  }
} 
```

This is probably applicable to more than just here though but might be a good pattern to set when we need to pass in an ID and text. I'm just kind of waffling here though, not saying you should change it here unless you've got time, I might can look into it next sprint since I have a couple cleanup related items on me.

Reply:
@LZoog This looks very nice, but I'm a bit low on time for this sprint. Could be good to consider making an adaptation of this that can also account for vars, attributes, elems? Just a thought.

Reply:
That's an interesting idea too, I think I'll bump up my card-header clean up ticket this sprint to a 2 pointer and look at some various cleanup.